src/app/compare/compare-tree/compare-tree.component.ts

```diff
@@ -40,15 +42,19 @@ export class CompareTreeComponent {
     highlightOpenFolders: false,
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
+    determineIconClass: SimpleFileTreeUtil.conditionalCssClass,
   };
 
   rightTreeOptions: FileTreeOptions = {
     highlightOpenFolders: false,
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
     determineFontColor: (item: CreateTreeItem) => this.setRedLabels(item),
+    determineIconClass: SimpleFileTreeUtil.conditionalCssClass,
   };
 
+  constructor(private helperService: HelperService) {}
```

Comment:
You can now also remove the helper service from the constructor

Reply:
added the change